File: Iniciante/URI 1010.cs
using System; 

class URI {

    public static void Main(string[] args) { 

        double result;
        
        string line1 = Console.ReadLine();
        string line2 = Console.ReadLine();
        
        string[] values1 = line1.Split(' ');
        int c1 = Int32.Parse(values1[0]);
        int n1 = Int32.Parse(values1[1]);
        double v1 = Convert.ToDouble(values1[2]);
        
        string[] values2 = line2.Split(' ');
        int c2 = Int32.Parse(values2[0]);
        int n2 = Int32.Parse(values2[1]);
        double v2 = Convert.ToDouble(values2[2]);
    
        result= (n1*v1) + (n2*v2);
        
        Console.WriteLine("VALOR A PAGAR: R$ {0:F2}", result);
    }
}

File: Iniciante/URI 1010.c
#include <stdio.h>

int main()
{

  int c1, c2, n1, n2;
  double v1, v2, result;

  scanf("%d %d %lf", &c1, &n1, &v1);
  scanf("%d %d %lf", &c2, &n2, &v2);

  result = (n1 * v1) + (n2 * v2);

  printf("VALOR A PAGAR: R$ %.2lf", result);
  printf("\n");

  return 0;
}